playground/AspireEventHub/EventHubs.ServiceDefaults/Extensions.cs

```diff
@@ -0,0 +1,116 @@
+using Microsoft.AspNetCore.Builder;
```

Comment:
Remove this and use [the shared service defaults](https://github.com/dotnet/aspire/tree/2788dd5a66ae4902f9d1a6a08c19f71359b7ea95/playground/Playground.ServiceDefaults)

Reply:
ah, I think I'm already referencing the shared playground project but I didn't notice this. thanks.